specification/appendix/commitment_discounts/size_flexibility.md

```diff
@@ -0,0 +1,200 @@
+# Size Flexibility
+
+A usage-based commitment discount obligates a customer to a usage amount for one or more related SKUs in return for reduced rates.  For example, when a customer purchases a usage-based commitment discount to cover a specific database SKU, this commitment will cover every hour over the term where at least one instance of this SKU is running. It can also cover eligible resources wholly, where 1 hour is covered by 1 resource, or partially, where 1 committed hour can be split across multiple resources or 1 resource can be fully covered with unused commitment discount costs. These latter two examples are commonly known as *size-flexibility*.
+
+Since providers have different rules for when size-flexibility is or is not enabled, the following, made-up SKU pricing attributes will be used in each example below, and each example will be categorized under size-flexibility enabled or disabled sections.
```

Comment:
```suggestion
Since providers have different rules for when size flexibility is or is not enabled, the following, fictitious SKU pricing attributes will be used in each example below, and each example will be categorized under size flexibility enabled or disabled sections.
```